src/components/textfield/textfield.ts

```diff
@@ -102,6 +104,15 @@ export class MTextfield extends ModulVue {
     private get propPasswordIcon(): boolean {
         return this.passwordIcon && this.type === MTextfieldType.Password && this.as<InputState>().active;
     }
+
+    private get hasWordWrap(): boolean {
+        let hasWordWrap: boolean = this.inputType === MTextfieldType.Text && this.wordWrap;
+        this.as<InputManagement>().trimWordWrap = hasWordWrap;
```

Comment:
Assigner une valeur sur un {{get}}, je trouve ça hasardeux même si dans les faits, ça fonctionne.

Reply:
Si on veut que `this.as<InputManagement>().trimWordWrap` soit modifié en conséquence de la valeur de `this.inputType` et `this.wordwarp`, j'irais tout simplement avec des watchs sur les deux props.